cmd/add.go

```diff
@@ -60,7 +60,6 @@ func rAdd(ccmd *cobra.Command, args []string) error {
 	}
 
 	if install {
-		initConfig()
```

Comment:
while talking to David, this actually needs to stay, as need to re-initialize config to pull the written changes back in.